src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,21 @@
+//card size
+$card-width: 200px;
+$card-margin-top: 46px;
+$card-margin-right: 48px;
```

Comment:
@huberciak your code seem proper while the deployed page has reversed margins:
![switched margins (top and right)](https://user-images.githubusercontent.com/39308717/160694460-12c6d606-8334-4322-9e88-7a987c55f155.png)
top 48 / right 46
make sure you deployed the most recent implementation - with edits applied test should pass.